enigma-core/enclave/Makefile

```diff
@@ -13,4 +19,12 @@ ifeq ($(XARGO_SGX), 1)
 else
 	cargo build $(CARGO_FLAGS)
 	cp ./target/$(Rust_target_dir)/libenigmacoreenclave.a ../lib/libenclave.a
-endif
\ No newline at end of file
+endif
+
+
+.PHONY: bindgen
+bindgen: Enclave_t.h
+	cargo build -p enigma-types # Meant to make sure `enigma-types.h` already exists and can be included.
+	echo $(BINDGEN_RAW_LINES)
+	bindgen Enclave_t.h $(BINDGEN_FLAGS) -- $(BINDGEN_CLANG_FLAGS) > $(BINDGEN_OUTPUT_FILE)
+	rustfmt  $(BINDGEN_OUTPUT_FILE)
```

Comment:
Why didn't you use rustfmt option of builder in  #165?

Reply:
I'll recheck but I'm pretty sure that it doesn't run on the added `raw_lines` which I add them all at the same line, so `rustfmt` will sort them later nicely